addons/spreadsheet/models/res_lang.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+from odoo.addons.spreadsheet.utils import odooLangToSpreadsheetLocale
+
+
+class Lang(models.Model):
+    _inherit = "res.lang"
+
+    @api.model
+    def get_locales_for_spreadsheet(self):
```

Comment:
? 
```suggestion
    def get_spreadsheet_locales(self):
```

Reply:
What I didn't like about this name is that it kinda implies that we are getting the locales _of_ a spreadsheet ...